include/vapor/FlowParams.h

```diff
@@ -53,35 +53,94 @@ class PARAMS_API FlowParams : public RenderParams {
 
     static std::string GetClassType() { return ("FlowParams"); }
 
-    // True  == Steady; False == Unteady
+    //! Sets the type of flow rendering algorithm being used.
+    //! Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
+    //! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
+    //! \param[in] bool - Steady/streamlines = 1, Unsteady/pathlines = 0
     void SetIsSteady(bool steady);
+    
+    //! Gets the type of flow rendering algorithm being used.
+    //! Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
+    //! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
+    //! \retval bool - Steady/streamlines = 1, Unsteady/pathlines = 0
     bool GetIsSteady() const;
 
+    //! Get the multiplier being applied to the flow advection algorithm.
+    //! If there happens to be a mismatch between the units of your data's domain and the units of a variable such as wind speed,
+    //! you can scale the wind field with this parameter.  IE - If your data's domain is written in kilometers but your wind
+    //! vectors are in meters, you can apply a velocity multiplyer of 1000 to correct the mismatch.
```

Comment:
To convert velocity in meters to kilometers don't you want to multiply by 0.001, not 1000.0?